core/src/replay_stage.rs

```diff
@@ -1684,7 +1785,7 @@ impl ReplayStage {
 
     #[allow(clippy::too_many_arguments)]
     pub(crate) fn compute_bank_stats(
-        my_pubkey: &Pubkey,
+        my_vote_pubkey: &Pubkey,
```

Comment:
The caller of `compute_bank_stats` should have changed from `my_pubkey` to `vote_account` as well, right?

Reply:
ugh yeah, good catch!